AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/assign-quotas-dialog/assign-quotas-dialog.component.ts

```diff
@@ -38,42 +39,31 @@ export class AssignQuotasDialogComponent {
         this.periods = [];
 
         this.loading = true;
+
+        this.errorMessage = "";
     }
 
     public ngOnInit(): void {
-        this.getMetrics().subscribe(() => {
-            if (this.data.assignQuotaData) {
-                for (const metric of this.metrics) {
-                    if (metric.key === this.data.assignQuotaData.metricKey) {
-                        this.metric = metric;
-                    }
-                }
-                this.max = this.data.assignQuotaData.max;
-                this.period = this.data.assignQuotaData.period;
-            }
-        });
+        this.getMetrics();
         this.getPeriods();
     }
 
-    public getMetrics(): Observable<any> {
+    public getMetrics(): void {
```

Comment:
Why is it called `get...` if it doesn't return anything?

Reply:
To be honest I am not sure.. the name of that method is not one of my doing.. I will rename it as "showMetrics"